msal/src/androidTest/java/com/microsoft/identity/client/PublicClientConfigurationTest.java

```diff
@@ -294,7 +293,7 @@ public void testUnknownAuthorityException() {
      */
     @Test(expected = IllegalArgumentException.class)
     public void testUnknownAudienceException() {
-        final PublicClientApplicationConfiguration configWithInvalidAudience = loadConfig(R.raw.test_pcaconfig_unknown_audience);
+        final PublicClientApplicationConfiguration configWithInvalidAudience = loadConfigAndMerge(R.raw.test_pcaconfig_unknown_audience);
```

Comment:
we might want to just replace loadConfig with loadConfigAndMerge in every function.